ansible/roles/operator-pipeline/templates/openshift/tasks/publish-to-ocp-registry.yml

```diff
@@ -27,6 +27,10 @@ spec:
         #! /usr/bin/env bash
         set -xe
 
+        dnf install jq -y
```

Comment:
Please include a comment about why this is necessary. It's unfortunate if we have to install dependencies like this. Is there at least an item in the backlog to track fixing this tech debt?